presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergUtil.java

```diff
@@ -856,10 +865,10 @@ public static long getDataSequenceNumber(ContentFile<?> file)
      * @param requestedSchema If provided, only delete files with this schema will be provided
      */
     public static CloseableIterable<DeleteFile> getDeleteFiles(Table table,
-                                                               long snapshot,
-                                                               TupleDomain<IcebergColumnHandle> filter,
-                                                               Optional<Set<Integer>> requestedPartitionSpec,
-                                                               Optional<Set<Integer>> requestedSchema)
+            long snapshot,
+            TupleDomain<IcebergColumnHandle> filter,
+            Optional<Set<Integer>> requestedPartitionSpec,
+            Optional<Set<Integer>> requestedSchema)
```

Comment:
```suggestion
            long snapshot,
            TupleDomain<IcebergColumnHandle> filter,
            Optional<Set<Integer>> requestedPartitionSpec,
            Optional<Set<Integer>> requestedSchema)
```

Can we revert the whitespace only changes?